### PR TITLE
Add event recorder to kernel controller

### LINKED ISSUE #495: Add event recorder to kernel chaos controller
## Feature Request

Kernel chaos controller should support event recorder as well just like other controllers.

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: yeya24 <yb532204897@gmail.com>

### What problem does this PR solve? 

Fixes #495

### CODE REVIEW

File: controllers/kernelchaos_controller.go [hunk 1]
```diff
@@ -15,6 +15,7 @@ package controllers
 
 import (
 	"github.com/go-logr/logr"
+	"k8s.io/client-go/tools/record"
```
Comment (cwen0) [inline controllers/kernelchaos_controller.go:18]: move this line to the last group

Comment (yeya24) [reply controllers/kernelchaos_controller.go:18]: Updated

**Pull request merged**